styles/components/_dropdown.scss

```diff
@@ -70,6 +70,17 @@
     background-color: $brand-gray-lt;
   }
 
+  > .menu > a .badge {
+    float: right; 
+    background-color: $brand-primary;
```

Comment:
This should be emerald, right? I think `.list-group` also extends `.badge.-emerald`. No?